Extensions/shuffle+.js

```diff
@@ -459,19 +459,39 @@
 	async function Queue(list, context = null, type) {
 		const count = list.length;
 
+		// Delimits the end of our list, as Spotify may add new context tracks to the queue
 		list.push("spotify:delimiter");
 
-		await Spicetify.Platform.PlayerAPI.clearQueue();
+		{
```

Comment:
Btw why do you execute it in separate block? There's no variable that may conflict with other ones I believe

Reply:
Visual clarity, The function is getting pretty long and I just figured that It would be nice to delimit that block

Reply:
since it's not needed, please remove it